packages/admin/src/pages/super-admin-dashboard/user/[id]/change-roles.test.tsx

```diff
@@ -68,4 +83,74 @@ describe('Edit role page', () => {
       screen.getByText('Applicant').parentElement?.previousSibling
     ).not.toBeChecked();
   });
+
+  test('Should redirect to change department page if new Admin user', async () => {
+    (parseBody as jest.Mock).mockReturnValue({ newUserRoles: ['3', '4'] });
+    (getUserById as jest.Mock).mockReturnValue({
```

Comment:
You can avoid casting these in each test with [jest.mocked](https://jestjs.io/docs/mock-function-api#jestmockedsource-options)

```js
// below jest.mock calls at the top of the file
const mockParseBody = jest.mocked(parseBody);
const mockGetUserById = jest.mocked(getUserById);
```

You get back the object you passed in but with the expected mock function properties, so then you can just do

```js
parseBody.mockReturnValue({ ... });
```

without the casting